R/http-libcurl.R

```diff
@@ -122,19 +105,10 @@ httpLibCurl <- function(protocol,
 
   # make the request
   response <- NULL
-  time <- system.time(gcFirst = FALSE, tryCatch({
-      # fetch the response into a raw buffer in memory
-      response <- curl::curl_fetch_memory(url, handle = handle)
-    },
-    error = function(e, ...) {
-      # ignore errors resulting from timeout or user abort
-      if (identical(e$message, "Callback aborted") ||
-          identical(e$message, "transfer closed with outstanding read data remaining"))
-        return(NULL)
-      # bubble remaining errors through
-      else
-        stop(e)
-    }))
+  time <- system.time(
```

Comment:
I also simplified this since the user interrupt error message has changed so that had no effect and I'm pretty sure that "transfer closed" (https://github.com/curl/curl/blob/37a3c63427e84a56703e19b6ab7c78fa52e566a2/lib/transfer.c#L1215-L1228) is a legit error we do want to bubble up.

Reply:
Ok, I'll have a go with some uploading some really large files and see if I can recreate it.

Reply:
I uploaded a 1 Gb app to shinyapps and Posit connect, and didn't see any problems. I'll follow up on the rpubs issue later since it's already individually tracked.

Reply:
I uploaded a 33 Mb html with no problems. You can see it in all its glory at https://rpubs.com/hadley/large-file.

Reply:
Also tested updating, since that uses a `PUT` instead of `POST`.

Reply:
Was that a large Rmd or a large bundle? Because we gzip, it's harder to get a large bundle. I ended up using a specific document when last playing with that issue, but it's likely changed since I tried.. I think I have a copy locally somewhere.  https://github.com/rstudio/rsconnect/issues/450#issuecomment-780114862

Reply:
@aronatkins From the progress bar 😄 ` Uploaded 23796573 bytes...` so ~22 Mb.